[FE] Add changesets (#4214)
MystenLabs/sui: .changeset/README.md

# Changesets

> Hello and welcome! This folder has been automatically generated by `@changesets/cli`, a build tool that works

Reviewer: ```suggestion
Hello and welcome! This folder was automatically generated by `@changesets/cli`, a build tool that works
```

with multi-package repos, or single-package repos to help you version and publish your code. You can
find the full documentation for it [in our repository](https://github.com/changesets/changesets)

> We have a quick list of common questions to get you started engaging with this project in

Reviewer: ```suggestion
You can view a list of [common questions](https://github.com/changesets/changesets/blob/main/docs/common-questions.md) to help you get started with changesets. 
```

> [our documentation](https://github.com/changesets/changesets/blob/main/docs/common-questions.md)

Reviewer: ```suggestion

```
